uqcsbot/scripts/yt.py

```diff
@@ -0,0 +1,54 @@
+import os
+from uqcsbot import bot, Command
+
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+YOUTUBE_API_KEY = os.environ.get('YOUTUBE_API_KEY')
+YOUTUBE_API_SERVICE_NAME = 'youtube'
+YOUTUBE_API_VERSION = 'v3'
+YOUTUBE_VIDEO_URL = 'https://www.youtube.com/watch?v='
+
+
+@bot.on_command('yt')
+def handle_yt(command: Command):
+    '''
+    `!yt <QUERY>` - Returns the top video search result based on the query string.
+    '''
+    # Makes sure the query is not empty.
+    if command.has_arg():
+        search_query = command.arg.strip()
+        try:
+            videoID = get_top_video_result(search_query, command.channel)
+        except HttpError as e:
```

Comment:
you should probably force `return` after logging to ensure the script doesn't continue onto returning to the user "Your query returned no results."